checks.d/php_fpm.py

```diff
@@ -0,0 +1,120 @@
+# 3p
+import requests
+
+# project
+from checks import AgentCheck
+from util import headers
+
+
+class PHPFPMCheck(AgentCheck):
+    """
+    Tracks basic php-fpm metrics via the status module
+    Requires php-fpm pools to have the status option.
+    See http://www.php.net/manual/de/install.fpm.configuration.php#pm.status-path for more details
+    """
+
+    SERVICE_CHECK_NAME = 'php_fpm.alive'
+
+
+    GAUGES = {
+        'listen queue': 'php_fpm.listen_queue',
+        'max listen queue': 'php_fpm.max_listen_queue',
+        'listen queue len': 'php_fpm.listen_queue_len',
+        'idle processes': 'php_fpm.idle_processes',
+        'active processes': 'php_fpm.active_processes',
+        'total processes': 'php_fpm.total_processes',
+        'max active processes': 'php_fpm.max_active_processes',
+        'max children reached': 'php_fpm.max_children_reached'
+    }
+    COUNTERS = {
+        'accepted conn': 'php_fpm.accepted_conn',
+        'slow requests': 'php_fpm.slow_requests'
+    }
+
```

Comment:
A few suggestions about these names:

* Group some of them (`processes.active`, `processes.total`, `processes.max_active`, ...)
* Make some of them more clear (what are listen_queue and listen_queue_len? Again, group them `listen_queue.something`)
* We could call prefix that `fpm.` instead of `php_fpm`
* Use service check names close to what we already have. (`is_connect` is common but maybe not relevant in the case, `is_running` sounds better and that's what we have with Gunicorn)